contentcuration/contentcuration/viewsets/file.py

```diff
@@ -73,17 +95,17 @@ class Meta:
             "contentnode",
             "assessment_item",
             "preset",
-            "duration"
+            "duration",
         )
         list_serializer_class = BulkListSerializer
 
 
 def retrieve_storage_url(item):
-    """ Get the file_on_disk url """
+    """Get the file_on_disk url"""
```

Comment:
Hm... seems my formatter got aggressive here?

Reply:
I think that this was applied by `pre-commit` - I just accepted whatever pre-commit did thinking that it rewrote things related to my particular changes and didn't realize it was linting the whole file.